packages/gatsby-plugin-graphql-config/package.json

```diff
@@ -1,18 +1,23 @@
 {
   "name": "gatsby-plugin-graphql-config",
   "description": "Gatsby plugin to write out a graphql-config with develop process endpoint configured",
-  "version": "1.0.0",
+  "version": "0.0.0",
```

Comment:
Is this for lerna?

Reply:
yes, I rather publish it under a 0.0.1 version rather a 1.0.0 for now.